util/reggen/validate.py

```diff
@@ -314,6 +314,7 @@ def bitmask(bfield):
     ],
     'alert_list': ['lnw', "list of peripheral alerts"],
     'regwidth': ['d', "width of registers in bits (default 32)"],
+    'hier_path': ['hp', "additional hierarchy path before the reg block instance"],
```

Comment:
I don't get this. Could you explain a little bit detail for this usage? What is the criteria to set this field value?

Reply:
OK. Looks like it is special handling for alert_handler I assume. As other modules follow the convention of u_reg format. I am not comfortable to have a field for a special module... Hum... Let me think about it more.

Reply:
yes, alert_handler has an extra wrap for u_reg `i_reg_wrap.u_reg`
Let me know if you have other approach for this. Thanks

Reply:
I don't have the best solution at this moment. What I feel is that putting the dv related & for a specific IP only into generic IP.hjson does't look aligned. 

But I understand the intention, as the value is used in reggen. Maybe we could add parameter to reggen when creating DV collateral.

Reply:
could you elaborate how to add parameter to reggen when creating DV collateral?

Reply:
actually I removed this line as it caused document render error and made CI fail.
Only the update in `gen_rtl.py` is needed.